mkdocs.yml

```diff
@@ -97,6 +97,13 @@ nav:
   - Home: index.md
   - Getting Started:
     - Tutorial: pages/tutorial.md
+  - Model Hub:
+    - pages/model_hub/index.md
+    - pages/model_hub/text_generation.md
+    - pages/model_hub/image_to_text.md
+    - pages/model_hub/asr.md
+    # - pages/model_hub/vad.md
```

Comment:
Is this for the batched wispher ?

Reply:
Yes, and it's not a generic. It's only suitable as a preprocessing step for batched whisper.  That's why I didn't add it to the model hub. If we add batched whisper we probably need to add a page for it or at least mention it in the Whisper deployment docs.